MinPlatformPkg/Include/Library/TestPointCheckLib.h

```diff
@@ -629,6 +629,86 @@ TestPointSmmExitBootServices (
   VOID
   );
 
+/**
+  This service verifies SMRR configuration at the End of DXE.
+
+  Test subject: SMRR.
+  Test overview: Verify SMRR is aligned and SMRR matches SMRAM_INFO.
+  Reporting mechanism: Set ADAPTER_INFO_PLATFORM_TEST_POINT_STRUCT.
+                       Dumps SMRR and SMRAM_INFO.
+
+  @retval EFI_SUCCESS         The test point check was performed successfully.
+  @retval EFI_UNSUPPORTED     The test point check is not supported on this platform.
+**/
+EFI_STATUS
+EFIAPI
+TestPointStandaloneMmEndOfDxeSmrrFunctional (
+  VOID
+  );
```

Comment:
Do you think these new Standalone MM APIs are required? Or could the API be MM neutral (like in this case `TestPointMmEndOfDxeSmrrFunctional()`)?

That way the test point check lib instance will implement the proper MM type (Traditional vs Standalone) but there's not duplicate APIs.

Reply:
Updated to consolidate duplicate code.